demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1662,7 +1662,7 @@ def get_modeling_rule_data(path: str, packs: Dict[str, Dict] = None):
     if (
         not id_ and "marketplacev2" in marketplaces
     ):  # TODO: Should be removed after we have an agreed id field for modeling rule
-        id_ = f"{pack}-{os.path.basename(path).split('.')[0]}"
+        id_ = f"{pack}-{Path(path).name.split('.')[0]}"
```

Comment:
```suggestion
        id_ = f"{pack}-{Path(path).stem}"
```
`stem` is the pure name, without the suffix